bin/approval_request_listener

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+require "bundler/setup"
+require "approval_request_listener"
+
+queue_host = ENV["QUEUE_HOST"] || "localhost"
```

Comment:
@syncrou can these 4 lines be wrapped into a function and reused  here and in config/initializers/approval_request_listener maybe as a class method

Reply:
In this case it would only be `queue_host` and `queue_port` which should be moved to a generic place where all environmental variables are set.  We have the same problem with the service order listener. I created this https://github.com/ManageIQ/catalog-api/issues/174 to handle that in a future PR.